Watchman.Discord/Areas/Protection/Services/MutingService.cs

```diff
@@ -72,7 +73,7 @@ private async Task NotifyUserAboutMute(Contexts contexts, UserContext mutedUser,
             {
                 {"Serwer:", server.Name},
                 {"Powód:", $"{muteEvent.Reason}"},
-                {"Czas wygaśnięcia:", TimeZoneInfo.ConvertTimeFromUtc(muteEvent.TimeRange.End, TimeZoneInfo.Local).ToString("dd.MM.yyyy HH:mm:ss", CultureInfo.InvariantCulture)}
+                {"Czas wygaśnięcia:", muteEvent.TimeRange.End.ToLocalTimeString()}
```

Comment:
a jaki czas to lokalny?
tzn domyślam sie że CEST
ale co gdyby serwer był w innej strefie czasowej

później trzeba będzie sie nad tym zastanowić
żeby też nie trzeba było przebudowywać całego serwera

Reply:
pod te czasy jest task
trzeba będzie to zrobić tak, aby każdy serwer mógł sobie ustawić swoją strefę i potem powiedzmy, że np. ta metoda ToLocalTimeString() pobierałaby czas danego serwera i odpowiednio go zwracała

Reply:
ok